e2e/maintenance.test.ts

```diff
@@ -0,0 +1,44 @@
+import { loginAsAdmin } from './test-util';
+import {
+  getNotificationDescriptionBox,
+  getNotificationMessageBox,
+} from './test-util-antd';
+import { test, expect } from '@playwright/test';
+
+test.beforeEach(async ({ page }) => {
+  await loginAsAdmin(page);
+  await page.getByRole('menuitem', { name: 'Maintenance' }).click();
+});
+
+test.describe('test maintenance page', () => {
+  test.describe('Recalculate Usage', () => {
+    test('click the Recalculate Usage button', async ({ page }) => {
+      await page.getByRole('button', { name: 'Recalculate Usage' }).click();
+
+      await expect(getNotificationMessageBox(page)).toContainText(
+        'Recalculate Usage',
+      );
+      // skip the Recalculating message because it's too fast
+      await expect(getNotificationDescriptionBox(page)).toContainText(
+        'Recalculation finished',
+      );
+    });
+  });
+
+  test.describe('Rescan Images', () => {
+    test('click the Rescan Images button', async ({ page }) => {
+      await page.getByRole('button', { name: 'Rescan Images' }).click();
+
+      await expect(getNotificationMessageBox(page)).toContainText(
+        'Rescan Images',
+      );
+
+      await expect(getNotificationDescriptionBox(page)).toContainText(
+        'Rescanning...',
+      );
+      await expect(getNotificationDescriptionBox(page)).toContainText(
```

Comment:
The rescan process can take longer than the default test timeout of 5 seconds, so this test can easily fail.

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/XqC2uNFuj0wg8I60sMUh/5902e831-3342-446a-b093-18e6382c0cd4.png)